tests/unit/requirements.txt

```diff
@@ -52,3 +52,6 @@ proxmoxer ; python_version > '3.6'
 #requirements for nomad_token modules
 python-nomad < 2.0.0 ; python_version <= '3.6'
 python-nomad >= 2.0.0 ; python_version >= '3.7'
+
+#requirements for github app lookup
+jwt
```

Comment:
I'm not sure, but why do you need this when you mock JWT related things in the unit tests above?

Reply:
oddly enough, I can't get mock to write the two import `jwk_from_pem` and `JWT` when jwt is not installed. 
I've added a workaround for this, so this removed.